Library/Config+Helpers.swift

```diff
@@ -2,8 +2,8 @@ import Foundation
 import KsApi
 
 extension Experiment.Name {
-  public func isEnabled(in _: Environment) -> Bool {
-    guard let experiments = AppEnvironment.current.config?.abExperiments else { return false }
+  public func isEnabled(in environment: Environment = AppEnvironment.current) -> Bool {
```

Comment:
I've noticed this function is not tested...can we add appropriate tests?

Reply:
This function stayed untested for over a year! 👀 I will add a test for it.